### PR TITLE
Add scrollbar to auto rules list, fixes #2514

### LINKED ISSUE #2514: Editor: Auto rules sometimes don't fit on screen
Should move them down or add a scroll bar. As reported by mind on Discord.
screenshot_2020-07-14_23-12-46

### PR DESCRIPTION
**Pull request by edg-l**
image

It doesn't show a scrollbar when there aren't more than 5 rules.

### CODE REVIEW

File: src/game/editor/popups.cpp [hunk 1]
```diff
@@ -1415,19 +1415,73 @@ static int s_AutoMapConfigCurrent = -100;
...
 	for(int i = -1; i < pAutoMapper->ConfigNamesNum(); ++i)
 	{
-		View.HSplitTop(2.0f, 0, &View);
-		View.HSplitTop(12.0f, &Button, &View);
-		if(pEditor->DoButton_MenuItem(&s_AutoMapperConfigButtons[i], i == -1 ? "None" : pAutoMapper->GetConfigName(i), i == s_AutoMapConfigCurrent, &Button, 0, 0))
-			s_AutoMapConfigSelected = i;
+		if(ListCur > ListStopAt)
+			break;
+
+		if(ListCur>= ListStartAt)
```
Comment (12pm) [inline src/game/editor/popups.cpp:1473]:
```suggestion
		if(ListCur >= ListStartAt)
```

**Comment by def-** [comment]
Scrolling with mousewheel in the popup should not zoom into the map. Might be a bit tricky to fix.

**Comment by edg-l** [comment]
Should be fixed.

**Comment by def-** [comment]
bors r+
Couldn't see an easier way to check if mouse is currently inside popup

**Pull request merged**